notebook.ipynb

```diff
@@ -2,7 +2,15 @@
  "cells": [
```

Comment:
<p>Need to spell out ERA5 and ORA5 acronyms on first mention.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Added acronyms section to header cell to cover this.</p>